community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/load_bq.py

```diff
@@ -282,6 +284,26 @@ def bq_submit(jobs):
     print(f"successfully loaded {len(jobs)} jobs")
 
 
+def batched_bq_submit(
+    jobs, submit_function=bq_submit, bq_row_batch_size=BQ_ROW_BATCH_SIZE
```

Comment:
No need for such complexity if only reason for this testability. Did you consider using mock patchers instead?
https://docs.python.org/3/library/unittest.mock.html#the-patchers

Reply:
yeah I think patching requires a bit of refactoring, currently the client etc are all global in load_bq